nise/yaml_gen.py

```diff
@@ -38,6 +46,15 @@ class DateRangeArgsError(Exception):
     pass
 
 
+def get_today_date():
+    return datetime.today().date()
+
+
+def get_last_month_start_date():
+    today = get_today_date()
+    return today.replace(day=1) - relativedelta(months=1)
```

Comment:
```suggestion
def get_last_month_start_date():
    today = datetime.today().date()
    return today.replace(day=1) - relativedelta(months=1)
```

There's no need to wrap stdlib functions if you're just returning the result unaltered.

Reply:
I used `get_today_date` in more than one spot. I think it's a little more explicit than `datetime.today().date()`